ui/src/overridableMapping.js

```diff
@@ -33,5 +34,6 @@ export const overriddenCmps = {
   'LoanMetadata.DeliveryIcon': DeliveryIcon,
   'RequestForPatronMessage.DeliveryIcon': DeliveryIcon,
   'LoanAvailability.NotAvailable': NotAvailable,
+  'Frontsite.route': LegacyDocumentRoute,
```

Comment:
```suggestion
  'Frontsite.route': LegacyRecordRoute,
```
since it can be also series

Reply:
Literature or Record, find with me! Let's just try to keep it consistent everywhere, backend and frontend